ams/server/src/main/java/com/netease/arctic/server/optimizing/OptimizingQueue.java

```diff
@@ -194,13 +196,17 @@ private void retryTask(TaskRuntime taskRuntime, boolean incRetryCount) {
 
   @Override
   public void ackTask(String authToken, int threadId, OptimizingTaskId taskId) {
+    OptimizerInstance optimizer = getAuthenticatedOptimizer(authToken);
+    LOG.debug("Optimizer executor[{},{}] ack task {}", optimizer.getToken(), threadId, taskId);
     Optional.ofNullable(executingTaskMap.get(taskId))
         .orElseThrow(() -> new TaskNotFoundException(taskId))
         .ack(new OptimizingThread(authToken, threadId));
   }
 
   @Override
   public void completeTask(String authToken, OptimizingTaskResult taskResult) {
+    OptimizerInstance optimizer = getAuthenticatedOptimizer(authToken);
+    LOG.debug("Optimizer executor[{},{}] complete task {}", optimizer.getToken(), taskResult.getThreadId(), taskResult.getTaskId());
```

Comment:
This log line is a bit redundant. It has already been printed in the outer layer.